### PR TITLE
[Test Case]: E2a-SP01

### PR DESCRIPTION
**Pull request by evandroAlfieri**
## Summary

Test Instructions for E2a-SP01: **(RFI) Track super-structure for single track**

## Purpose
Add Track super-structure test to the Track Exchange

### Test Cases

- [x] Creation of new test case
- [x] Test case summary
- [x] Itemised roots & test case imports
- [x] Usages, constraints & logic
- [x] Validation criteria & Results
- [x] Dataset Content & Documentation

@czapplitec , @larswik , @peterrdf can you have a look (at the whole instructions) in particular at:
- Material association
- Usages for geometry (i.e., Body SectionedSolidHorizontal for IfcCourse) --> hear I need input
- Sleeper placement

### CODE REVIEW

File: E2a-TRAS/SP01/README.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+These concept templates represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD, that must be correctly exported to meet the validation criteria. **The scope of the test shall not be used as a definitive scope of the exchange, or of the MVD**
+
+- Object Attributes
+   - Object Predefined Type
+- Object definition
+   - Object Typing
+- Product Shape
+   - Product Geometric Representation
+   - Body SectionedSolidHorizontal
```
Comment (czapplitec) [inline E2a-TRAS/SP01/README.md:58]: Body Tessallation Geometry
Product Local Placement
Product Linear Placement

File: E2a-TRAS/SP01/README.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+- Object Composition
+   - Spatial Decomposition
+- Object Connectivity
+   - Spatial Containment
+- Object Association
+   - Material Association
```
Comment (czapplitec) [inline E2a-TRAS/SP01/README.md:68]: What specific requirements for material are needed here. I could see the possibility to apply:
- Materal Single
- Material Profile Set
- Material Constituent Set

File: E2a-TRAS/SP01/README.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+| Spatial Element | Spatial Element Type | Minimum | Maximum | Element            | Element Type      |
+|-----------------|----------------------|---------|---------|--------------------|-------------------|
+| IfcFacilityPart | TRACKSTRUCTURE       | 1       |         | IfcCourse          | BALLASTBED        |
+| IfcFacilityPart | TRACKSTRUCTURE       | 2       |         | IfcRail            | RAIL              |
+| IfcFacilityPart | TRACKSTRUCTURE       | 1       |         | IfcTrackElement    | SLEEPER           |
+
+NOTE:
+- when **Maximum is empty**, it means **unlimited**. Example: Minimum=1; Maximum=empty, means that the Spatial Element must contain 1 or more elements of the requested type.
+
+
```
Comment (czapplitec) [inline E2a-TRAS/SP01/README.md:280]: Could you maybe make a suggested hierachy using bullet point structure to understand the required breakdown?

File: E2a-TRAS/SP01/README.md (hunk 1 above)
Comment (evandroAlfieri) [reply E2a-TRAS/SP01/README.md:58]: Added

File: E2a-TRAS/SP01/README.md (hunk 2 above)
Comment (evandroAlfieri) [reply E2a-TRAS/SP01/README.md:68]: added

File: E2a-TRAS/SP01/README.md (hunk 3 above)
Comment (evandroAlfieri) [reply E2a-TRAS/SP01/README.md:280]: done

**Pull request merged**